apps/dashboard/app/apps/manifest.rb

```diff
@@ -65,6 +65,8 @@ def initialize(opts)
     raise InvalidContentError.new unless(opts && opts.respond_to?(:to_h))
 
     @manifest_options = opts.to_h.with_indifferent_access
+
+    @manifest_options.reject! { |method| !self.respond_to?(method) }
```

Comment:
does select read better here than reject not?
```suggestion
    @manifest_options = opts.to_h.select do |method| 
       self.respond_to?(method.to_sym)
     end.with_indifferent_access
```

Reply:
Agreed that reads much cleaner and makes sense to chain the `with_indifferent_access` to the end here.

Reply:
Trying to use `with_indifferent_access` chained on the end causes errors for almost all the test cases with a message of `NoMethodError: undefined method `with_indifferent_access' for nil:NilClass` but keeping it on the original `to_h` results in everything passing with the `select!` method. 

Why does that failure happen? Because if no changes are made, then the `select!` will simply return `nil` which is what is causing the failures https://ruby-doc.org/core-2.7.2/Hash.html#method-i-select-21

So we are a bit stuck having that `with_indifferent_access` on the original `to_h` I believe.

Reply:
Got it. I'll update the comment with what may work.

Reply:
Yeah, that link was to the `select!` method which I was using (noticed the `select` wasn't what we wanted before). 

It notes `keep_if` is an alias for `select!`. Either way even using `select!` causes the behavior as it will still return a `nil` if nothing does change.